BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Group/GroupListModalView.js

```diff
@@ -0,0 +1,102 @@
+import OrchestraView        from '../OrchestraView'
+import ModalView            from '../../../Service/Modal/View/ModalView'
+import UserGroups           from '../../Collection/Group/UserGroups'
+import GroupListForUserView from './GroupListForUserView'
+import Application          from '../../Application'
+
+/**
+ * @class GroupListModalView
+ */
+class GroupListModalView extends ModalView
+{
+    /**
+     * @inheritdoc
+     */
+    preinitialize() {
+        this.tagName = 'div';
+        this.className = 'modal fade';
```

Comment:
this.tagName and this.className are already defined in modal view

use super.preinitialize